syntax = "proto3";

import "google/protobuf/timestamp.proto";

// A voicemail message.
message Voicemail {
    string to_number = 1;
    string from_number = 2;
    string audio_url = 3;
    string transcription = 4;
    google.protobuf.Timestamp timestamp = 5;
}